src/string.hh

```diff
@@ -83,6 +83,8 @@ public:
     StringView substr(CharCount from, CharCount length = INT_MAX) const;
     StringView substr(ColumnCount from, ColumnCount length = INT_MAX) const;
 
+    bool starts_with(StringView prefix) const;
```

Comment:
I dont like adding that to the string interface, it should be a free function in string_utils.hh.

Reply:
Actually this function has already been implemented, it's called `prefix_match()`!